src/sbt-test/skip-java17+/scalafixResolvers/project/plugins.sbt

```diff
@@ -1,3 +1,3 @@
 resolvers += Resolver.sonatypeRepo("public")
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % sys.props("plugin.version"))
-addSbtPlugin("au.com.onegeek" %% "sbt-dotenv" % "2.1.146")
+addSbtPlugin("au.com.onegeek" %% "sbt-dotenv" % "3.1.1")
```

Comment:
the groupId has changed some years ago to `nl.gn0s1s`: https://github.com/Philippus/sbt-dotenv/?tab=readme-ov-file#installation

Reply:
yep, just saw it as well, thanks for looking into it here!